assets/app/game_manager.rb

```diff
@@ -157,5 +157,6 @@ def update_game(game)
     @games.reject! { |g| g['id'] == game['id'] } if game['deleted']
     @games.map! { |g| g['id'] == game['id'] ? game : g }
     store(:games, @games.sort_by { |g| g['id'] }.reverse)
+    store(:game, game, skip: true) if @game && @game['id'] == game['id']
```

Comment:
put this before the previous store for convention

if @game.&['id']== game['id']